modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -0,0 +1,431 @@
+		_groupName = groupName;
+	}
+
+	public void setHorizontalCard(HorizontalCard horizontalCard) {
+		_horizontalCard = horizontalCard;
+	}
+
+	public void setHref(String href) {
+		_href = href;
+	}
+
+	public void setIcon(String icon) {
+		_icon = icon;
+	}
+
+	public void setInputName(String inputName) {
+		_inputName = inputName;
+	}
+
+	public void setInputValue(String inputValue) {
+		_inputValue = inputValue;
+	}
+
+	public void setSelectable(Boolean selectable) {
+		_selectable = selectable;
+	}
+
+	public void setSelected(Boolean selected) {
+		_selected = selected;
+	}
+
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+	 */
+	@Deprecated
+	public void setSpritemap(String spritemap) {
+		_spritemap = spritemap;
+	}
+
+	public void setTitle(String title) {
+		_title = title;
+	}
+
+	@Override
+	protected void cleanUp() {
+		super.cleanUp();
+
+		_actionDropdownItems = null;
+		_data = null;
+		_disabled = null;
+		_groupName = null;
+		_horizontalCard = null;
+		_href = null;
+		_icon = null;
+		_inputName = null;
+		_inputValue = null;
+		_selectable = null;
+		_selected = null;
+		_spritemap = null;
+		_title = null;
+	}
+
+	@Override
+	protected String getHydratedModuleName() {
+		return "frontend-taglib-clay/HorizontalCard";
+	}
+
+	@Override
+	protected Map<String, Object> prepareProps(Map<String, Object> props) {
+		props.put("actions", getActionDropdownItems());
+		props.put("disabled", isDisabled());
+		props.put("href", getHref());
+		props.put("inputName", getInputName());
+		props.put("inputValue", getInputValue());
+		props.put("selectable", isSelectable());
+		props.put("selected", isSelected());
+		props.put("symbol", getIcon());
+		props.put("title", getTitle());
+
+		return super.prepareProps(props);
+	}
+
+	@Override
+	protected String processCssClasses(Set<String> cssClasses) {
+		if (isSelectable()) {
+			cssClasses.add("card-type-directory");
+			cssClasses.add("form-check");
+			cssClasses.add("form-check-card");
+			cssClasses.add("form-check-middle-left");
+		}
+		else {
+			cssClasses.add("card");
+		}
+
+		return super.processCssClasses(cssClasses);
+	}
+
+	@Override
+	protected int processStartTag() throws Exception {
+		super.processStartTag();
+
+		JspWriter jspWriter = pageContext.getOut();
+
+		if (isSelectable()) {
+			jspWriter.write("<div class=\"custom-control custom-checkbox\">");
+			jspWriter.write("<label><input");
+
+			if (isSelected()) {
+				jspWriter.write(" checked");
+			}
+
+			if (isDisabled()) {
+				jspWriter.write(" disabled");
+			}
+
+			String inputName = getInputName();
+
+			if (inputName != null) {
+				jspWriter.write(" name=\"");
+				jspWriter.write(inputName);
+				jspWriter.write("\"");
+			}
+
+			String inputValue = getInputValue();
+
+			if (inputValue != null) {
+				jspWriter.write(" value=\"");
+				jspWriter.write(inputValue);
+				jspWriter.write("\"");
+			}
+
+			jspWriter.write(" class=\"custom-control-input\" type=\"");
```

Comment:
Sorting. `class` should be printed before `disabled` and after `checked`, and `type` before `value`